clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -73,6 +74,10 @@
  * functionality for accessing configs.
  */
 public class ConfigDef {
+
+    /** Statically compiled pattern for finding commas with potentially unlimited whitespace before and/or after the comma. */
```

Comment:
nit: comments like this seem redundant. Can we just remove them?

Reply:
If that's what you want, I'll remove them. My reason for adding was that it wasn't immediately obvious to me what the regex did. So I went to regex101.com and it explained the details, which I then added. It might be redundant if you're more fluent at regex than me, but it wasn't for me.

Reply:
Also, it's more for other people than for me. I want the people to come after me to be able to read this and find use out of it, instead of having to google it. I already did that, I might as well save the people after me the trouble.

Reply:
@hachikuji Just to clarify, I'm waiting on your confirmation that you still want it gone after reading my reasoning.

Reply:
@KoenDG I agree with @hachikuji. Name of the constant should be explanatory enough (and it usually is). I don't see the need to explain usage further.